yb-voyager/src/namereg/namereg.go

```diff
@@ -290,6 +288,19 @@ schema1.foobar, schema1."foobar", schema1.FooBar, schema1."FooBar", schema1.FOOB
 (fuzzy-case-match) schema1.fooBar, schema1."fooBar"
 */
 func (reg *NameRegistry) LookupTableName(tableNameArg string) (sqlname.NameTuple, error) {
+
+	sourceObjectNameMap := reg.SourceDBTableNames
+	targetObjectNameMap := reg.YBTableNames
+	if len(lo.Keys(reg.SourceDBSequenceNames)) > 0 {
+		for s, objs := range reg.SourceDBSequenceNames {
+			sourceObjectNameMap[s] = append(sourceObjectNameMap[s], objs...)
```

Comment:
I think this will overwrite reg.SourceDBTableNames. a map is just a pointer I believe, so even if you copy, when you modify an entry in the map, it will update the underlying map structure.

https://go.dev/play/p/xN9dzHoBSAT